src/mode.js

```diff
@@ -69,13 +80,19 @@ function getMode_() {
       // from the URL.
       location.originalHash || location.hash);
 
+  if (!fullVersion) {
+    fullVersion = getFullVersion_(window, isLocalDev);
+  }
+
   return {
     localDev: isLocalDev,
     // Triggers validation
     development: developmentQuery['development'] == '1' || window.AMP_DEV_MODE,
     minified: process.env.NODE_ENV == 'production',
     test: window.AMP_TEST,
     log: developmentQuery['log'],
+    version: version,
```

Comment:
Lets remove this. We version lock the child frame, so it can read this itself, and we need to be careful with the URL length.